.github/workflows/fpm-deployment.yml

```
@@ -31,32 +32,84 @@ jobs:
           version: ${{ matrix.toolchain.version }}
 
       - name: Setup Fortran Package Manager
-        uses: fortran-lang/setup-fpm@v5
+        uses: fortran-lang/setup-fpm@v7
```

The latest version is `v8`
```suggestion
        uses: fortran-lang/setup-fpm@v8
```